conf/defaults.ini

```diff
@@ -193,12 +193,19 @@ reporting_enabled = true
 reporting_distributor = grafana-labs
 
 # Set to false to disable all checks to https://grafana.com
-# for new versions (grafana itself and plugins), check is used
-# in some UI views to notify that grafana or plugin update exists
+# for new versions of grafana. The check is used
+# in some UI views to notify that a grafana update exists.
 # This option does not cause any auto updates, nor send any information
-# only a GET request to https://grafana.com to get latest versions
+# only a GET request to https://grafana.com to get latest versions.
```

Comment:
Not true. It does a request to https://raw.githubusercontent.com/grafana/grafana/main/latest.json. 

Maybe we want to change this later to actually use grafana.com, e.g. https://grafana.com/api/grafana/versions/stable and/or https://grafana.com/api/grafana/versions/beta.

Reply:
Dang it I thought I changed that :rage4: 

Good catch!